server/chat-plugins/repeats.ts

```diff
@@ -136,15 +136,10 @@ export const pages: PageTable = {
 		html += `<table><tr><th>${this.tr`Identifier`}</th><th>${this.tr`Phrase`}</th><th>${this.tr`Raw text`}</th><th>${this.tr`Interval`}</th><th>${this.tr`Action`}</th>`;
 		for (const repeat of room.settings.repeats) {
 			const minutes = repeat.interval / (repeat.isByMessages ? 1 : 60 * 1000);
-			if (!repeat.faq) {
-				const phrase = repeat.isHTML ? repeat.phrase : Chat.formatText(repeat.phrase, true);
-				html += `<tr><td>${repeat.id}</td><td>${phrase}</td><td>${Chat.getReadmoreCodeBlock(repeat.phrase)}</td><td>${repeat.isHTML ? this.tr`every ${minutes} chat message(s)` : this.tr`every ${minutes} minute(s)`}</td>`;
-				html += `<td><button class="button" name="send" value="/removerepeat ${repeat.id},${room.roomid}">${this.tr`Remove`}</button></td>`;
-			} else {
-				const phrase = Chat.formatText(roomFaqs[room.roomid][repeat.id], true);
-				html += `<tr><td>${repeat.id}</td><td>${phrase}</td><td>${Chat.getReadmoreCodeBlock(roomFaqs[room.roomid][repeat.id])}</td><td>${repeat.isHTML ? this.tr`every ${minutes} chat message(s)` : this.tr`every ${minutes} minute(s)`}</td>`;
-				html += `<td><button class="button" name="send" value="/removerepeat ${repeat.id},${room.roomid}">${this.tr`Remove`}</button></td>`;
-			}
+			const faqOrNot = repeat.faq ? roomFaqs[room.roomid][repeat.id] : repeat.phrase;
```

Comment:
`faqOrNot` sounds like a boolean, whereas this actually is a string. Maybe call it `repeatText` or something?

Reply:
did